assets/queries/cloudFormation/filter_and_alarm_exist_for_changes_to_NACL/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "Filter_And_Alarm_Exist_For_Changes_To_NACL",
+  "queryName": "Filter And Alarm Not Exists For Changes To NACL",
```

Comment:
Change **directory name** and **id** to: `nacl_changes_without_alarms`
Change **query name** to: `NACL Changes Without Alarms`